src/main/scala/org/opennetworkinsight/dns/DNSPostLDA.scala

```diff
@@ -24,15 +23,10 @@ object DNSPostLDA {
                  sqlContext: SQLContext,
                  logger: Logger) = {
 
-
     logger.info("DNS post LDA starts")
 
-    val topics = sc.broadcast(ipToTopicMixes)
-    val words = sc.broadcast(wordToProbPerTopic)
-
-
     val totalDataDF = {
-      sqlContext.parquetFile(inputPath.split(",")(0))
+      sqlContext.read.parquet(inputPath.split(",")(0))
```

Comment:
No problem. I'm utilizing IntelliJ VersionControl feature. From VersionControl view I get a confirmation window before committing and pushing changes where it tells me all the warnings in the code, the parquetFile appeared as "will be deprecated in 2.0", that's how I did this change.